lighthouse-core/report/html/report-styles.css

```diff
@@ -786,12 +786,13 @@
   border: 0;
 }
 
-.lh-scores-header .lh-gauge__wrapper,
-.lh-scores-header .lh-gauge--pwa__wrapper {
+.lh-scores-header .lh-gauge__wrapper {
   margin: 0 4px;
 }
 
 .lh-scores-header .lh-gauge--pwa__wrapper {
+  margin-right: 4px;
+  padding-left: 3px; /* 4px - 1px border */
```

Comment:
why do we need to subtract the border? didn't we have 9px of space before with margin-left (4) + margin-right (4) + border (1)

Reply:
Yeah it is currently a 9px space.  I removed the 1px just to make it 4x4 even because the symmetry is pleasing.  It's fair to make it 4x4 + 1px border 🤷‍♂️ I'm just brining up the concern.

Reply:
maybe my CSS math is just off :) but isn't having only 3px of padding making the border spacing asymmetrical?  or are you referring to consistent spacing between the category scores?

Reply:
It makes the overall spacing symmetrical with the rest of the gauges.  Or at least that was my intention.